core/sql/optimizer/BindItemExpr.cpp

```diff
@@ -10851,6 +10851,96 @@ ItemExpr *ZZZBinderFunction::bindNode(BindWA *bindWA)
       }
     break;
 
+    case ITM_CSV_FORMAT:
+      {
+	bindChildren(bindWA);
+	if (bindWA->errStatus()) 
+	  return this;
+
+        // The way the arguments of CSV_FORMAT are represented in
+        // the parse tree is as a tree of ItemList nodes; so
+        // CSV_FORMAT(a,b,c,d) is represented as
+        //
+        //    this
+        //    /  \
+        //   a   ItemList
+        //         /  \
+        //        b   ItemList
+        //              /  \
+        //             c    d
+        //
+        // The code below traverses accordingly.
```

Comment:
There is a class ItemExprList that can convert such trees into a collection class. Other binder methods use that to traverse lists.